spec/binarySearchTree.js

```diff
@@ -0,0 +1,55 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import BinarySearchTree from '../src/binarySearchTree'
+
+chai.use(chaiChange)
+
+describe('BinarySearchTree', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(BinarySearchTree).to.be.a('function')
+  })
+
+  context('insert()', () => {
+    it('inserts a node with the specified value into the tree.', () => {
+      const bst = new BinarySearchTree()
+
+      bst.insert(1)
+      bst.insert(2)
+      bst.insert(3)
+
+      expect(bst.count).to.eql(2)
+    })
+  })
+
+  context('search()', () => {
+    it('returns a node object.', () => {
+      const bst = new BinarySearchTree()
+
+      bst.search(10)
+      bst.search(15)
+      bst.search(20)
+
+      expect(bst.count).to.eql(0)
+    })
```

Comment:
the tests for lines 26-35 are not clear. Can you run the bst.search function inside of the expect and see if it returns that node?